### PR TITLE
Start to fill in trigger/reco working group page

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

File: _workinggroups/recotrigger.md [hunk 1]
```diff
@@ -20,6 +22,16 @@ effective use of modern computing hardware through threading,
 vectorisation and use of accelerator technologies; and the application
 of profiling and quality assurance toolkits.
 
-[Mandate](/organization/working-group-mandates.html).
+## Get involved
+
+Current WG convenors: Caterina Doglioni, Agnieszka Dziurda, David Lange. [Contact the group convenors](mailto:caterina.doglioni@cern.ch,agnieszka.dziurda@cern.ch,david.lange@cern.ch)
```
Comment (graeme-a-stewart) [inline _workinggroups/recotrigger.md]: You could do with a full stop at the end there.

File: _workinggroups/recotrigger.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: "Reconstruction and Software Triggers"
 layout: plain
 ---
 
+## Mandate and Goals
+
 The Reconstruction and Software Triggers Working Group will consider
```
Comment (graeme-a-stewart) [inline _workinggroups/recotrigger.md]: will consider -> considers

File: _workinggroups/recotrigger.md [hunk 3]
```diff
@@ -20,6 +22,16 @@ effective use of modern computing hardware through threading,
...
+## Get involved
+
+Current WG convenors: Caterina Doglioni, Agnieszka Dziurda, David Lange. [Contact the group convenors](mailto:caterina.doglioni@cern.ch,agnieszka.dziurda@cern.ch,david.lange@cern.ch)
+
+Everyone is welcome to participate and contribute on the forum and to the ongoing meetings. For more information, contact us or
+follow our mailing list on google groups [hsf-recotrigger](https://groups.google.com/forum/#!forum/hsf-recotrigger).
+
+## Group activities
+
+* Reconstruction+Trigger sessions at HOW2019 JLab include discussions on real-time analysis, reconstruction on accelerators, and more: [Agenda](https://indico.cern.ch/event/759388/timetable/#20190320.detailed)
```
Comment (graeme-a-stewart) [inline _workinggroups/recotrigger.md]: Agenda -> agenda
And add a full stop.

**Comment by davidlange6** [comment]
thanks @graeme-a-stewart - I addressed these.

**Pull request merged**